### DISCUSSION TITLE
Get User List as Json from Sqlite db?

### DISCUSSION THREAD

**Discussion by hsul-git** [fastapi-users/fastapi-users#1089] [Q&A] [answered]
Hey everyone,

iam trying to implement a route for a superuser to get a list of all users (id, email, is_activ, is_superuser, is_verified) from a sqlite db using SQLAlchemy. I tried to rewrite some examples for mongodb, but it didnt worked out, because it kinda confused me a little how it actually works.

Do you guys have a hint for me?

Thank you.

**Comment by frankie567** [comment, marked as answer]
Can be something like this:

```py
from sqlalchemy import select


@app.get("/users", response_model=list[UserRead])
async def list_users(session: AsyncSession = Depends(get_async_session)):
    statement = select(User)
    result = await session.execute(statement)
    return result.scalars().all()
```

Basically, it's just a query using the SQLAlchemy ORM: https://docs.sqlalchemy.org/en/14/orm/extensions/asyncio.html#synopsis-orm

**Comment by hsul-git** [comment]
Thanks, that helped a lot. Thumbs up.
